extensions-contrib/druid-iceberg-extensions/pom.xml

```diff
@@ -271,6 +272,22 @@
       <artifactId>iceberg-hive-metastore</artifactId>
       <version>${iceberg.core.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.iceberg</groupId>
+      <artifactId>iceberg-aws</artifactId>
+      <version>${iceberg.core.version}</version>
+    </dependency>
+<!--    GlueCatalog class-->
+    <dependency>
+      <groupId>software.amazon.awssdk</groupId>
+      <artifactId>bundle</artifactId>
```

Comment:
Could we replace this uber dependency with just the required AWS client dependencies for Glue? The bundle jar seems to add another 590MB to the extension.

Reply:
I will add just aws SDK Glue, I think that is sufficient.